### PR TITLE
Populate cp_db_version on new installations

### PR DESCRIPTION
**Pull request by xxsimoxx**
See #1552 .

## Description
In 2.2.0 `$cp_db_version` was introduced.
But for new installations it needs also to be in `schema.php`

## Types of changes
- Bug fix

### CODE REVIEW

**Comment by xxsimoxx** [comment]
Easy way to test:
- Clone the repo `git clone git@github.com:ClassicPress/ClassicPress.git`
- Spin up a new installation pointing MAMP or similar to `/src` folder
- Notice that "Database Update Required" page is loaded on the first login
- Remove `wp-config.php` and empty the database
- Checkout this PR: `gh pr checkout 1553`
- Spin up a new installation pointing MAMP or similar to `/src` folder
- No "Database Update Required" page is loaded

File: src/wp-admin/includes/schema.php [hunk 1]
```diff
@@ -580,6 +580,9 @@ function populate_options( array $options = array() ) {
 
 		// 6.4.0
 		'wp_attachment_pages_enabled'     => 0,
+
+		// CP-2
```
Comment (mattyrob) [inline src/wp-admin/includes/schema.php:584]: Can this be extended to CP-2.3.0and we can get it into core for the next minor release.

**Review by mattyrob — CHANGES_REQUESTED** [review]

File: src/wp-admin/includes/schema.php (hunk 1 above)
Comment (xxsimoxx) [reply src/wp-admin/includes/schema.php:584]: Wondering if there will be a 2.2.1.

Comment (mattyrob) [reply src/wp-admin/includes/schema.php:584]: Next version is probably for discussion at the next core meeting, leave this for now until we can discuss and decide.

Comment (xxsimoxx) [reply src/wp-admin/includes/schema.php]: Moved to CP-2.3.0.

**Review by mattyrob — APPROVED** [review]

**Pull request merged**